### PR TITLE
EUI-4549 Angular upgrade to 7..2.13

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4549

### Change description ###
Angular upgrade to 7.2.13

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
@johnbenjamin-hmcts  great job. We finally made CCD-Ui toolkit Angular upgrade [long pending]

File: demo/package.json [hunk 1]
```diff
@@ -12,23 +12,23 @@
...
-    "@angular/forms": "~7.0.3",
-    "@angular/http": "~7.0.3",
-    "@angular/material": "^7.0.3",
-    "@angular/platform-browser": "^7.0.3",
-    "@angular/platform-browser-dynamic": "^7.0.3",
-    "@angular/router": "~7.0.3",
-    "@angular/upgrade": "~7.0.3",
+    "@angular/animations": "^7.2.13",
+    "@angular/cdk": "^7.3.7",
+    "@angular/cli": "^6.0.8",
```
Comment (Daniel-Lam) [inline demo/package.json:20]: @johnbenjamin-hmcts - quick query on the apparent downgrade of `@angular/cli` version... is this intentional?

File: package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "prebuild": "rimraf dist tmp",
     "prebuild:watch": "rimraf dist tmp",
     "preversion": "npm run ci",
-    "test": "ng test --code-coverage --no-watch --no-progress --browsers=ChromeHeadlessCI --source-map=true",
+    "test": "ng test --code-coverage --no-watch --no-progress --browsers=ChromeHeadlessCI",
```
Comment (Daniel-Lam) [inline package.json:37]: @johnbenjamin-hmcts the `--source-map=true` bit has been removed...?

File: package.json [hunk 3]
```diff
@@ -46,13 +46,13 @@
     "heroku-postbuild": "yarn build:esm && yarn --cwd demo install && yarn --cwd demo build",
     "demo": "yarn && yarn build:esm && yarn build:demo && yarn start",
     "build:test:functional": "yarn && yarn build:esm && yarn --cwd demo  test:functional",
-    "test:functional":"yarn --cwd demo  test:functional"
+    "test:functional": "yarn --cwd demo  test:functional"
```
Comment (Daniel-Lam) [inline package.json:49]: @johnbenjamin-hmcts - might as well take out the extra space between `demo` and `test` :)

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - just two queries and one observation; the rest is fine.

File: package.json [hunk 4]
```diff
@@ -46,13 +46,13 @@
     "heroku-postbuild": "yarn build:esm && yarn --cwd demo install && yarn --cwd demo build",
     "demo": "yarn && yarn build:esm && yarn build:demo && yarn start",
     "build:test:functional": "yarn && yarn build:esm && yarn --cwd demo  test:functional",
```
Comment (Daniel-Lam) [inline package.json:48]: @johnbenjamin-hmcts - might as well take out the extra space between `demo` and `test` :)

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - spotted one more thing :)

File: package.json (hunk 2 above)
Comment (johnbenjamin-hmcts) [reply package.json:37]: Hi @Daniel-Lam , yes had to remove due to the following error:
_PS C:\HMCTS\ccd-case-ui-toolkit> yarn test
yarn run v1.22.11
$ ng test --code-coverage --no-watch --no-progress --browsers=ChromeHeadlessCI --source-map=true
Schema validation failed with the following errors:
  Data path ".sourceMap" should be object.
  Data path ".sourceMap" should be boolean.
  Data path ".sourceMap" should match exactly one schema in oneOf.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command._

The tests are running fine without this param, and MC doesn't have this defined as well. So I had removed it.

File: package.json (hunk 3 above)
Comment (johnbenjamin-hmcts) [reply package.json:49]: Thank you @Daniel-Lam, running the command `yarn install` auto-corrected the spacing after `:` but not the other one :)

File: package.json (hunk 4 above)
Comment (johnbenjamin-hmcts) [reply package.json:48]: Thank you @Daniel-Lam, I have now updated the code.

File: package.json (hunk 2 above)
Comment (Daniel-Lam) [reply package.json:37]: @johnbenjamin-hmcts - ah, OK... no worries, that's fine.

File: demo/package.json (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply demo/package.json:20]: Hi @Daniel-Lam, I tried to keep CCD UI Toolkit in sync with MC, and in MC the version of `@angular/cli` is 6.0.8

Comment (Daniel-Lam) [reply demo/package.json:20]: @johnbenjamin-hmcts - oh, OK... fair enough. At least it's consistent with MC :)

**Review by Daniel-Lam — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Pull request closed without merging**